presto-product-tests/bin/run_on_docker.sh

```diff
@@ -167,6 +168,16 @@ if [[ "$ENVIRONMENT" == "multinode" ]]; then
    PRESTO_SERVICES="${PRESTO_SERVICES} presto-worker"
 fi
 
+CLI_ARGUMENTS="--server presto-master:8080"
```

Comment:
I checked that. The way you provide/override environment variables in `compose-commons` and environmental `compose.sh` doesn't fit for me. Variables declared in those files are visible from the `compose.yaml`, but not from the main scripts. In `run_on_docker.sh` we run `compose.sh` in a new process, but not with the `source`. So all the exported variables can't be visible in a parent shell.